Makefile

```diff
@@ -310,27 +309,45 @@ dockerpull:
 
 .PHONY: test
 test: guard-VENV clean_logs local-templates build
-	@echo "${GREEN}Unit testings...${RESET}";
+	mkdir -p junit-reports/integration
+	@echo "${GREEN}Unit testing (integration tests)...${RESET}";
+	${PYTHON} ./scripts/pg_ready.py
+	${NOSE} \
+	-c tests/nose2_tests_integration.cfg \
+	-s tests/integration \
+	--verbose
+	-N 7
```

Comment:
NB process to 7 might not be a good choice for every one ! This needs to be set by an ENVIRONMENT variable and the default to `0` which means that nose2 set it to the actual number of cpu

Reply:
Good point, thanks! The `N` is more a relict of my local testing, since I wanted to make that faster.
Might be good to leave it parallelized tough, but as you say, using an env var for it makes sense :+1: